src/test/java/com/exasol/rls/administration/AbstractAdminScriptIT.java

```diff
@@ -54,14 +55,31 @@ protected static void initialize(final ExasolContainer<? extends ExasolContainer
         script = schema.getScript(scriptName);
     }
 
-    protected abstract Connection getConnection() throws NoDriverFoundException, SQLException;
+    public AbstractAdminScriptIT() {
+        try {
+            this.connection = EXASOL.createConnection("");
+        } catch (NoDriverFoundException | SQLException exception) {
+            throw new RuntimeException(exception);
+        }
+    }
+
+    @Override
+    protected void finalize() {
```

Comment:
Finalize is deprecated. Can't you use AfterAll or closable here?

Reply:
I reduced all connections to a single static one. The instance-based option was an artifact of the Java variant anyway where three different connection types needed to be tested. This is not relevant in the Lua variant.